packages/flutter/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## [3.2.0](https://github.com/parse-community/Parse-SDK-Flutter/compare/V3.1.1...flutter-3.2.0) (2022-06-22)
+
+### Bug Fixes
```

Comment:
Needs correction, see my comment in the dart changelog.

Reply:
In a next version, I can do the split changelog per package.
Release this version and the next ones we will do as requested.

Reply:
We are already doing this since the last releases, this is already a hard requirement for PRs.

Reply:
Done